File: app/src/main/java/com/limayeneha/twitter_trov/data/TweetsDatabase.java
package com.limayeneha.twitter_trov.data;
import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Created by limayeneha on 5/27/17.
 */

@Database(name = TweetsDatabase.NAME, version = TweetsDatabase.VERSION)
public class TweetsDatabase {
    public static final String NAME = "TweetsDatabase";

    public static final int VERSION = 1;
}

File: app/src/main/java/com/limayeneha/twitter_trov/data/TweetsDatabase.kt
package com.limayeneha.twitter_trov.data

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Created by limayeneha on 5/27/17.
 */

@Database(name = TweetsDatabase.NAME, version = TweetsDatabase.VERSION)
object TweetsDatabase {
    const val NAME: String = "TweetsDatabase"

    const val VERSION: Int = 1
}
